Documentation/AdditionalFunctionality/Jobs.md

```diff
@@ -0,0 +1,132 @@
+![document type: vision](https://apodini.github.io/resources/markdown-labels/document_type_vision.svg)
+
+# Jobs
+
+`Job`s allow to schedule repeating tasks and are in contrast to `Component`s not request based.
+Use cases for `Job`s could be running background database operations or fetching data from a remote server.
+In Apodini such tasks are defined by conforming to the `Job` protocol which includes a `run` method.
+This method is executed at scheduled points in time or when triggered by an [ObservableObject](<./../Communicational\ Patterns/2.\ Tooling/2.4.\ ObservedObject.md>).
+
+## Usage
+
+`Job`s can include the `@Environment` property wrappers to use other services in the `run` method.
```

Comment:
I would maybe just state that `Job`s can use all property wrappers that `Handler`s can also use that are not directly connected to an incoming request. But e.g. `@State` and `@ObservedObject` also would be possible.